Kitodo/src/main/java/org/kitodo/production/forms/CommentTooltipView.java

```diff
@@ -68,4 +68,28 @@ public List<Comment> getComments(ProcessDTO processDTO) {
     public List<Comment> getComments(TaskDTO taskDTO) {
         return getComments(taskDTO.getProcess());
     }
+
+    /**
+     * Get the most recent comment of the given process.
+     *
+     * @param processDTO process as ProcessDTO
+     * @return message of the comment
+     */
+    public String getLastComment(ProcessDTO processDTO) {
+        List<Comment> comments = getComments(processDTO);
+        if (comments.isEmpty()) {
+            return "";
+        }
+        return comments.get(0).getMessage();
```

Comment:
You are sure that `get(0)` is returning the last created comment? [CommentDAO.getAllByProcess](https://github.com/kitodo/kitodo-production/blob/master/Kitodo-DataManagement/src/main/java/org/kitodo/data/database/persistence/CommentDAO.java#L53-L56) is returning a list of comments ordered by ascending comment id and as the comment ids are increasing by adding new comment entries the `get(0)` call will return the oldest created entry.

Reply:
Thank you for pointing out! This was a copy paste error, since I developed this feature in a branch where the comment order in the tooltip is reversed (most recent first). I added another commit to retrieve the last entry from the list.

Reply:
As this whole class did not have any tests this could break again if your other branch with the reversed ordering get merged. Maybe you are remember to change it here again but if could even get forgotten.